flink-clients/src/main/java/org/apache/flink/client/deployment/ClusterSpecification.java

```diff
@@ -67,8 +68,8 @@ public String toString() {
 	public static ClusterSpecification fromConfiguration(Configuration configuration) {
 		int slots = configuration.getInteger(TaskManagerOptions.NUM_TASK_SLOTS, 1);
 
-		int jobManagerMemoryMb = configuration.getInteger(JobManagerOptions.JOB_MANAGER_HEAP_MEMORY);
-		int taskManagerMemoryMb = configuration.getInteger(TaskManagerOptions.TASK_MANAGER_HEAP_MEMORY);
+		int jobManagerMemoryMb = (int) MemorySize.parse(configuration.getString(JobManagerOptions.JOB_MANAGER_HEAP_MEMORY)).getMebiBytes();
```

Comment:
hi @dawidwys this belongs suggestion 2, "change the return value of getMebiBytes() to int or have a getMebiBytesAsInt() method that uses a MathUtils.checkedDownCast() to avoid unnoticed overflow errors, as Stephan commented"

"have a getMebiBytesAsInt()" is another choice, if I picked the first choice and change the `(int) to  MathUtils.checkedDownCast()` . Is there necessary to provide the `getMebiBytesAsInt()` method?